Index: app/src/main/java/com/example/vaio/learnmoregame/MainActivity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/example/vaio/learnmoregame/MainActivity.java	(date 1480277513000)
+++ app/src/main/java/com/example/vaio/learnmoregame/MainActivity.java	(revision )
@@ -22,6 +22,7 @@
 import android.support.v7.widget.RecyclerView;
 import android.support.v7.widget.SearchView;
 import android.support.v7.widget.Toolbar;
+import android.util.Log;
 import android.view.Gravity;
 import android.view.Menu;
 import android.view.MenuInflater;
@@ -29,26 +30,31 @@
 import android.view.View;
 import android.view.Window;
 import android.view.WindowManager;
+import android.widget.AdapterView;
 import android.widget.LinearLayout;
 import android.widget.ListView;
 import android.widget.Toast;
 
 
+import com.example.vaio.adapter.ListViewAdapter;
 import com.example.vaio.adapter.ListViewDrawerLayoutAdapter;
 import com.example.vaio.adapter.ViewPagerAdapter;
 import com.example.vaio.database.MyDatabase;
 import com.example.vaio.dialog.IntroductionDialog;
+import com.example.vaio.fragment.BaseFragment;
+import com.example.vaio.model_object.ItemListView;
 import com.example.vaio.parser.JsoupParser;
 
 import java.util.ArrayList;
 
-public class MainActivity extends AppCompatActivity implements TabLayout.OnTabSelectedListener, MenuItem.OnMenuItemClickListener {
+public class MainActivity extends AppCompatActivity implements TabLayout.OnTabSelectedListener, MenuItem.OnMenuItemClickListener, AdapterView.OnItemClickListener {
 
     public static final String ACTION_GAME = "Action";
     public static final String FPS_GAME = "FPS";
     public static final String OPEN_WORLD_GAME = "Open world";
     public static final String SURVIVAL_GAME = "Survival";
     public static final String TPS_GAME = "TPS";
+    private static final String TAG = "MainActivity";
     private Toolbar toolbar;
     private TabLayout tabLayout;
     private ViewPager viewPager;
@@ -57,32 +63,44 @@
     private ActionBarDrawerToggle drawerToggle;
     private SearchView searchView;
     private LinearLayout linearLayoutListMain;
+    private LinearLayout linearLayoutListChosseFromDrawerLayout;
     private IntroductionDialog introductionDialog;
 
-    private MyDatabase database;
+
     private ListViewDrawerLayoutAdapter listViewDrawerLayoutAdapter;
     private ArrayList<Integer> arrCountContentDrawerLayout;
     private RecyclerView listViewDrawerLayout;
 
+    //thao tac khi thuc hien drawer layout
+    private MyDatabase database;
+    private ListView lvChosseFromDrawerLayout;
+    private ListViewAdapter lvChosseAdapter;
+    private ArrayList<ItemListView> arrItemListViews;
 
+
     @RequiresApi(api = Build.VERSION_CODES.LOLLIPOP)
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
-        database = new MyDatabase(this);
         //khoi tao
         arrCountContentDrawerLayout = new ArrayList<>();
+        database = new MyDatabase(this);
         initCount();
         listViewDrawerLayoutAdapter = new ListViewDrawerLayoutAdapter(arrCountContentDrawerLayout);
-
         initToolbar();
+
+        //
+        arrItemListViews = new ArrayList<>();
+        lvChosseAdapter = new ListViewAdapter(this, arrItemListViews);
+
         initMainViews();
 
     }
 
     //khoi tao dem so tren drawerlayout
     private void initCount() {
+        arrCountContentDrawerLayout.add(0);
         arrCountContentDrawerLayout.add(12);
         arrCountContentDrawerLayout.add(0);
     }
@@ -98,6 +116,7 @@
 
     @RequiresApi(api = Build.VERSION_CODES.LOLLIPOP)
     private void initMainViews() {
+        //set title action bar
 
         final Window window = getWindow();
         window.addFlags(WindowManager.LayoutParams.FLAG_DRAWS_SYSTEM_BAR_BACKGROUNDS);
@@ -105,6 +124,7 @@
         window.setStatusBarColor(ContextCompat.getColor(getBaseContext(), R.color.black));
 
         linearLayoutListMain = (LinearLayout) findViewById(R.id.linear_list_main);
+        linearLayoutListChosseFromDrawerLayout = (LinearLayout) findViewById(R.id.fragment_list_drawerlayout);
         //drawerLayout
         drawerLayout = (DrawerLayout) findViewById(R.id.drawerLayout);
         drawerToggle = new ActionBarDrawerToggle(this, drawerLayout, R.string.app_name, R.string.app_name) {
@@ -146,17 +166,33 @@
         listViewDrawerLayout.setLayoutManager(horizontalLayoutManagaer);
         listViewDrawerLayout.setAdapter(listViewDrawerLayoutAdapter);
         listViewDrawerLayoutAdapter.setOnItemClickListener(clickDrawerLayout);
+
+        //listview chon tu drawerlayout
+        lvChosseFromDrawerLayout = (ListView) findViewById(R.id.lv_chosse_drawerlayout);
+        lvChosseFromDrawerLayout.setAdapter(lvChosseAdapter);
+        lvChosseFromDrawerLayout.setOnItemClickListener(this);
     }
 
+    @Override
+    public void onItemClick(AdapterView<?> adapterView, View view, int i, long l) {
+        Intent intent = new Intent(this, ContentGameActivity.class);
+        intent.putExtra(BaseFragment.KEY_INTENT_CHANGE, arrItemListViews.get(i));
+        startActivity(intent);
+    }
+
     private ListViewDrawerLayoutAdapter.OnItemClickListener clickDrawerLayout = new ListViewDrawerLayoutAdapter.OnItemClickListener() {
         @Override
         public void onItemClick(View view, int position) {
             switch (position) {
                 case 0:
+                    linearLayoutListMain.setVisibility(View.VISIBLE);
+                    linearLayoutListChosseFromDrawerLayout.setVisibility(View.INVISIBLE);
                     break;
                 case 1:
+                    chooseDrawerLayout(1);
                     break;
                 case 2:
+                    chooseDrawerLayout(2);
                     break;
                 case 3:
                     break;
@@ -194,6 +230,24 @@
         }
     };
 
+    //cap nhat listview khi chon item trong drawerlayout
+    //action hanh dong cua item tren drawerlayout
+    //1:xem,2:thich
+    private void chooseDrawerLayout(int action) {
+        linearLayoutListMain.setVisibility(View.INVISIBLE);
+        linearLayoutListChosseFromDrawerLayout.setVisibility(View.VISIBLE);
+        if (action == 1) {
+            arrItemListViews.clear();
+            arrItemListViews.addAll(database.getDataFromGameTable(MyDatabase.TB_NAME_LIST_LATER));
+            lvChosseAdapter.notifyDataSetChanged();
+        } else if (action == 2) {
+            arrItemListViews.clear();
+            arrItemListViews.addAll(database.getDataFromGameTable(MyDatabase.TB_NAME_LIST_LIKE));
+            Log.e(TAG,arrItemListViews.size()+"");
+            lvChosseAdapter.notifyDataSetChanged();
+        }
+    }
+
     public static boolean isNetworkAvailable(Context context) {
         final ConnectivityManager connectivityManager = ((ConnectivityManager) context.getSystemService(Context.CONNECTIVITY_SERVICE));
         return connectivityManager.getActiveNetworkInfo() != null && connectivityManager.getActiveNetworkInfo().isConnected();
@@ -250,4 +304,6 @@
         }
         return false;
     }
+
+
 }
Index: app/build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/build.gradle	(date 1480277513000)
+++ app/build.gradle	(revision )
@@ -2,7 +2,7 @@
 
 android {
     compileSdkVersion 25
-    buildToolsVersion "25.0.0"
+    buildToolsVersion "24.0.3"
     defaultConfig {
         applicationId "com.example.vaio.learnmoregame"
         minSdkVersion 15
@@ -51,9 +51,4 @@
     compile 'com.android.support:design:25.0.0'
     compile 'org.jsoup:jsoup:1.10.1'
     compile 'com.squareup.picasso:picasso:2.5.2'
-    compile 'com.android.support:appcompat-v7:25.0.0'
-    compile 'com.android.support:design:25'
-    compile 'com.android.support:design:25.0.0'
-    compile 'com.android.support:recyclerview-v7:25.0.1'
-    compile 'com.android.support:design:25.0.1'
 }
Index: app/src/main/res/layout/activity_main.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/res/layout/activity_main.xml	(date 1480277513000)
+++ app/src/main/res/layout/activity_main.xml	(revision )
@@ -9,7 +9,6 @@
     tools:context="com.example.vaio.learnmoregame.MainActivity">
 
     <LinearLayout
-        android:id="@+id/linear_list_main"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
         android:orientation="vertical">
@@ -32,29 +31,49 @@
                 android:orientation="horizontal"></LinearLayout>
         </android.support.v7.widget.Toolbar>
 
+        <RelativeLayout
+            android:layout_width="match_parent"
+            android:layout_height="match_parent">
+
-        <LinearLayout
+            <LinearLayout
-            android:id="@+id/linear_fragment_main"
+                android:id="@+id/linear_list_main"
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:orientation="vertical">
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:orientation="vertical">
 
-            <android.support.design.widget.TabLayout
-                android:id="@+id/tabLayout"
-                android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                android:background="@color/colorPrimary"
-                android:minHeight="?attr/actionBarSize"
-                android:theme="@style/ThemeOverlay.AppCompat.Dark.ActionBar"
-                app:tabGravity="center"
-                app:tabMode="scrollable" />
+                <android.support.design.widget.TabLayout
+                    android:id="@+id/tabLayout"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:background="@color/colorPrimary"
+                    android:minHeight="?attr/actionBarSize"
+                    android:theme="@style/ThemeOverlay.AppCompat.Dark.ActionBar"
+                    app:tabGravity="center"
+                    app:tabMode="scrollable" />
 
-            <android.support.v4.view.ViewPager
-                android:id="@+id/viewPager"
-                android:layout_width="match_parent"
-                android:layout_height="wrap_content" />
-        </LinearLayout>
+                <android.support.v4.view.ViewPager
+                    android:id="@+id/viewPager"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content" />
+            </LinearLayout>
 
+            <LinearLayout
+                android:id="@+id/fragment_list_drawerlayout"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:background="#3dbde7"
+                android:orientation="vertical"
+                android:visibility="invisible">
+
+                <ListView
+                    android:id="@+id/lv_chosse_drawerlayout"
+                    android:layout_width="match_parent"
+                    android:layout_height="match_parent"></ListView>
-    </LinearLayout>
+            </LinearLayout>
+        </RelativeLayout>
+
+    </LinearLayout>
+
 
     <LinearLayout
         android:layout_width="match_parent"
Index: .idea/inspectionProfiles/Project_Default.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- .idea/inspectionProfiles/Project_Default.xml	(date 1480277513000)
+++ .idea/inspectionProfiles/Project_Default.xml	(revision )
@@ -1,7 +1,6 @@
 <component name="InspectionProjectProfileManager">
   <profile version="1.0">
     <option name="myName" value="Project Default" />
-    <inspection_tool class="AndroidLintNotSibling" enabled="false" level="ERROR" enabled_by_default="false" />
     <inspection_tool class="AndroidLintValidFragment" enabled="false" level="ERROR" enabled_by_default="false" />
     <inspection_tool class="LoggerInitializedWithForeignClass" enabled="false" level="WARNING" enabled_by_default="false">
       <option name="loggerClassName" value="org.apache.log4j.Logger,org.slf4j.LoggerFactory,org.apache.commons.logging.LogFactory,java.util.logging.Logger" />
\ No newline at end of file
Index: build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- build.gradle	(date 1480277513000)
+++ build.gradle	(revision )
@@ -6,6 +6,7 @@
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:2.2.2'
+
         // NOTE: Do not place your application dependencies here; they belong
         // in the individual module build.gradle files
     }
